sdk/tables/azure-data-tables/tests/test_table_entity.py

```diff
@@ -2236,3 +2238,21 @@ def test_upsert_entity_with_invalid_key_type(self, tables_storage_account_name,
                 self.table.upsert_entity(entity2)
         finally:
             self._tear_down()
+
+    @tables_decorator
+    @recorded_by_proxy
```

Comment:
The recordings for these tests are empty because the service request never gets made, so we could save space in the repo by not recording these tests. The fact that these tests could run without recordings is evidence that service calls aren't being made, so I don't think there's a reason to keep the empty recordings